system/HTTP/URI.php

```diff
@@ -787,51 +775,54 @@ protected function filterPath(string $path=null)
 		$path = $this->removeDotSegments($path);
 
 		// Fix up some leading slash edge cases...
-		if (strpos($orig, './') === 0) $path = '/'. $path;
-		if (strpos($orig, '../') === 0) $path = '/'. $path;
+		if (strpos($orig, './') === 0) $path	 = '/'.$path;
+		if (strpos($orig, '../') === 0) $path	 = '/'.$path;
 
 		// Encode characters
 		$path = preg_replace_callback(
-			'/(?:[^' . self::CHAR_UNRESERVED . ':@&=\+\$,\/;%]+|%(?![A-Fa-f0-9]{2}))/',
-			function (array $matches)
-			{
-				return rawurlencode($matches[0]);
-			},
-			$path
+			'/(?:[^'.self::CHAR_UNRESERVED.':@&=\+\$,\/;%]+|%(?![A-Fa-f0-9]{2}))/', function (array $matches)
+		{
+			return rawurlencode($matches[0]);
+		}, $path
 		);
 
 		return $path;
 	}
 
 	//--------------------------------------------------------------------
 
-
-
 	/**
 	 * Saves our parts from a parse_url call.
 	 *
 	 * @param $parts
 	 */
 	protected function applyParts($parts)
 	{
-		if (! empty($parts['host'])) $this->host = $parts['host'];
-		if (! empty($parts['user'])) $this->user = $parts['user'];
-		if (! empty($parts['path'])) $this->path = $this->filterPath($parts['path']);
-		if (! empty($parts['query'])) $this->query = $this->filterQuery($parts['query']);
-		if (! empty($parts['fragment'])) $this->fragment = $this->filterQuery($parts['fragment']);
+		if ( ! empty($parts['host'])) $this->host		 = $parts['host'];
+		if ( ! empty($parts['user'])) $this->user		 = $parts['user'];
+		if ( ! empty($parts['path'])) $this->path		 = $this->filterPath($parts['path']);
+		if ( ! empty($parts['query']))
+				$this->query	 = $this->filterQuery($parts['query']);
+		if ( ! empty($parts['fragment']))
+				$this->fragment	 = $this->filterQuery($parts['fragment']);
 
```

Comment:
Agreed, maybe. PSR-2 dictates curly braces, but does not address line wrapping, as far as I can see. Our style guide does not appear to be explicit for control structures, other than saying that we follow "Allman style", and I have come up with conflicting interpretations of that :(
Lonnie & I are trying to sort out the style guide, and my NetBeans settings.
What a pain in the ***!